nms/app/packages/magmalte/app/views/events/EventsTable.js

```diff
@@ -250,6 +330,21 @@ export default function EventsTable(props: EventTableProps) {
   function DateFilter() {
     return (
       <Grid container justify="flex-end" alignItems="center" spacing={1}>
+        <Grid item>
+          <FormControlLabel
```

Comment:
Make this as a separate component and move it under components. You can perhaps have this and the useAutoRefresh hook in the same file.